pkg/restore/crd_v1_preserve_unknown_fields_action.go

```diff
@@ -101,3 +106,18 @@ func (c *CRDV1PreserveUnknownFieldsAction) Execute(input *velero.RestoreItemActi
 		UpdatedItem: &unstructured.Unstructured{Object: res},
 	}, nil
 }
+
+func fromUnstructured(unstructured map[string]interface{}) (*apiextv1.CustomResourceDefinition, error) {
```

Comment:
Would it make sense to put this function in a shared location, since the JSON marshaling is also done on the back up side?

Reply:
I can do that if you prefer; I considered it but didn't bother since it's such a small function. If you want me to do that, any particular package you have in mind? `pkg/util/kube` is really the only one that comes to mind.

Reply:
I don't think it's a blocker for this PR - I could move it in mine, too.

I also agree that it would probably end up in that package.